kcidb/db/__init__.py

```diff
@@ -2,14 +2,14 @@
 
 import sys
 import logging
-import argparse
 import datetime
 import kcidb.io as io
 import kcidb.orm
 import kcidb.misc
 from kcidb.misc import LIGHT_ASSERTS
-from kcidb.db import abstract, schematic, mux, \
-    bigquery, postgresql, sqlite, json, null, misc  # noqa: F401
+from kcidb.db import abstract, schematic, mux, argparse, \
+     bigquery, postgresql, sqlite, json, null, misc  # noqa: F401
+from .argparse import QueryArgumentParser
```

Comment:
Could you avoid importing this separately, and just use `argparse.QueryArgumentParser`?